lib/sensors/low_power_current.cpp

```diff
@@ -0,0 +1,44 @@
+#include <low_power_current.hpp>
+
+namespace hyped::sensors {
+
+// TODO: Implement failure state for create
+std::optional<LowPowerCurrent> LowPowerCurrent::create(core::ILogger &logger,
+                                                       std::shared_ptr<io::II2c> i2c,
+                                                       const std::uint8_t channel
+                                                       = kDefaultI2cAddress)
```

Comment:
dont do default here, check it in `create` itself, also `defaultLowPowerCurrentAddress`

Reply:
also maybe call `channel` - `device_address`

Reply:
Isn't `kDefaultLowPowerCurrentAddress` a bit long? Mind you, not sure what would be a suitably shorter yet equally clear substitute

Reply:
@kshxtij said do it to be more clear